sippy-ng/src/component_readiness/CompReadyMainInputs.js

```diff
@@ -80,8 +83,44 @@ export default function CompReadyMainInputs(props) {
       ></AdvancedOptions>
     </div>
   )
+
+  // eslint-disable-next-line react/prop-types
+  function ViewPicker({ enabled }) {
+    if (!enabled) return null
+    return (
+      <div className={classes.crRelease}>
+        <FormControl variant="standard">
```

Comment:
View gets very small when it's empty, this should make it look a little better

```suggestion
        <FormControl variant="standard" sx={{ minWidth: 100 }}>
```

Reply:
Done